lds/rv64/task-link.x

```diff
@@ -76,6 +76,30 @@ SECTIONS
   .eh_frame (INFO) : { KEEP(*(.eh_frame)) }
   .eh_frame_hdr (INFO) : { *(.eh_frame_hdr) }
 
+  .debug_loc (INFO) : { *(.debug_loc) }
```

Comment:
Just curious: Do we need to explicitly list out each section here? My understanding was that if a section isn't defined in the linker script, then that section would anyways be created in the order in which it appears in the object file. You just lose control over where to place them.

Reply:
There are occasionally bugs that can occur from missing sections, like when we forgot `sbss` or `sdata`. I think doing this along with turning on errors for orphaned sections is a good idea.

Reply:
Yeah, listing out orphaned sections can be helpful from catching any missing sections. Though I don't think we would have too many new sections going forward. But, my curiosity was along the lines of what Drew asked.. how does this help/improve what existed without this PR.

Reply:
This PR normalises rv64 and rv32 linker scripts as we already list out these individual sections in rv32.

Reply:
> This PR normalises rv64 and rv32 linker scripts as we already list out these individual sections in rv32.

In that case, I think we should put that in the commit message to ensure that the motivation for the change is captured.

Reply:
Individual commit messages updated